src/include/user/tdfb.h

```diff
@@ -31,14 +33,36 @@
 struct sof_tdfb_config {
 	uint32_t size;			/* Size of entire struct */
 	uint16_t num_filters;		/* Total number of filters */
-	uint16_t num_output_channels;    /* Total number of output channels */
+	uint16_t num_output_channels;   /* Total number of output channels */
 	uint16_t num_output_streams;	/* one source, N output sinks */
 	uint16_t reserved16;		/* To keep data 32 bit aligned */
```

Comment:
push this 16-bit reserved down to the 32-bit reserved area, it's more usable like that

Reply:
Done